voluncheer/profiles/views/gallery.py

```diff
@@ -0,0 +1,41 @@
+from django.shortcuts import redirect
+from django.shortcuts import render
+
+from profiles.forms.gallery import CreateGalleryPostForm
+from profiles.models import Volunteer
+
+
+def create_post(request):
+    """create a new Post and save it to the database."""
+    user = request.user
+    if user.is_anonymous:
+        return redirect("home")
+    if user.is_volunteer:
```

Comment:
What will happen if an organization tries to access the form?

Reply:
Good point! I'll add a guard here